static/app/views/relocation/relocation.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import Relocation from 'sentry/views/relocation/relocation';
+
+describe('Relocation', function () {
+  function renderPage(step) {
+    const routeParams = {
+      step,
+    };
+
+    const {routerProps, routerContext, organization} = initializeOrg({
+      router: {
+        params: routeParams,
+      },
+    });
+
+    render(<Relocation {...routerProps} />, {
+      context: routerContext,
+      organization,
+    });
+  }
+  describe('Get Started', function () {
+    it('renders', async function () {
+      renderPage('get-started');
+      expect(
+        await screen.findByText('Basic Information Needed to Get Started')
+      ).toBeInTheDocument();
+      expect(
+        await screen.findByText(
+          'In order to best facilitate the process some basic information will be required to ensure sucess with the relocation process of you self-hosted instance'
+        )
+      ).toBeInTheDocument();
+      expect(await screen.findByText('Organization Slugs')).toBeInTheDocument();
+      expect(await screen.findByText('Choose a Datacenter Region')).toBeInTheDocument();
```

Comment:
Btw, I remember we had some scary warnings in our original design, that made it super clear this is a very consequential decision. @jmanhart do you have suggestions on what to do here?

Reply:
oh shoot, I missed the part where the warning appears only when a data region is selected, I'll go through that again

Reply:
implemented